src/scenes/DeliverablesRouter/QuestionsDeliverableView.tsx

```diff
@@ -187,9 +188,11 @@ const QuestionsDeliverableView = (props: Props): JSX.Element | null => {
                 paddingTop: theme.spacing(3),
               }}
             >
-              {variablesWithValues.map((variableWithValues: VariableWithValues, index: number) => (
-                <QuestionBox key={index} projectId={projectId} variable={variableWithValues} />
-              ))}
+              {variablesWithValues.map((variableWithValues: VariableWithValues, index: number) =>
+                variableDependencyMet(variableWithValues, variablesWithValues) ? (
```

Comment:
Should we validate this on the external side too?

Reply:
So this view is for the external (participant) side, but still a good question. I'm not sure if there's a requirement to conditionally render the questions on the internal (accelerator) side.

Reply:
I'll make another PR for the internal side

Reply:
Do we need to conditionally render questions still on the internal side? I could see the case for internal users wanting to see all of the questions, regardless of the current answer/value status.

Reply:
Good question, I'll check to see if there's a clear answer in the PRD